lib/ui/painting/image_decoder_impeller.cc

```diff
@@ -162,8 +163,9 @@ std::optional<DecompressResult> ImageDecoderImpeller::DecompressTexture(
   const auto pixel_format =
       impeller::skia_conversions::ToPixelFormat(image_info.colorType());
   if (!pixel_format.has_value()) {
-    FML_DLOG(ERROR) << "Codec pixel format not supported by Impeller.";
-    return std::nullopt;
+    std::string decode_error("Codec pixel format not supported by Impeller.");
```

Comment:
Can we include anything here about what the color type was listed as according to Skia?

Reply:
I'd get rid of the Impeller bit. There aren't pixel formats that are unique to either Skia or Impeller that are supported by Flutter.